warehouse/query-core/src/main/java/datawave/query/iterator/FieldIndexOnlyQueryIterator.java

```diff
@@ -253,7 +254,18 @@ protected void createAndSeekIndexIterator(Range range, Collection<ByteSequence>
         Collection<String> unindexedTypes = Lists.newArrayList();
         
         Set<String> keys = fetchDatatypeKeys(this.documentOptions.get(NON_INDEXED_DATATYPES));
-        unindexedTypes.addAll(keys);
+        
+        String compressedOptionString = this.documentOptions.get(QUERY_MAPPING_COMPRESS);
```

Comment:
Where are we setting these options in a compressed manner.  I would have expected to see compression being done somewhere in concert with this decompression.